index.js

```diff
@@ -713,6 +713,12 @@
   //. Type comprising every primitive String value.
   var String_ = NullaryTypeWithUrl('String', typeofEq('string'));
 
+
+  //# Symbol :: Type
+  //.
+  //. Type comprising every primitive Symbol value.
```

Comment:
An artifact of copying. There is no boxed version of `Symbol` thus no need to differentiate it.

Though it is a primitive.